### DISCUSSION TITLE
Sesto senso 2 support?

### DISCUSSION THREAD

**Discussion by xfred81** [indilib/indi#1819] [Q&A] [answered]
Hello!

I'd like to buy a focuser and I've been suggested the Sesto Senso 2.
Yet, I only found the "v1" within supported devices and with a "moderate" rating.

Could someone confirm me is Senso 2 is supported and behaves correctly on Indi? (I only use Linux/Kstars/EKOS/Indi).

Thanks a lot!

**Comment by knro** [comment, marked as answer]
It's well supported.

**Reply by xfred81** [reply]
Thanks a lot. You use it already and see it working on Linux/Indi?

I mean, PrimeLuceLab just answered my email and claims "in order to properly works SESTO SENSO 2 can be installed in the EAGLE or an external computer running Windows 7, 8, 10 or 11 operating system.", and indicates that with Linux to make Senso 2 to work is "to install a virtual machine and install Windows.".

**Reply by knro** [reply]
Yes it works fine on Linux.

**Reply by xfred81** [reply]
Thanks a lot! ;) I saw you contributed on this part... so thanks twice!

**Reply by xfred81** [reply]
Another detail, if you don't mail. Do you know if Kstars has a "calibration" interface to let the Senso 2 working smoothly? Or wouldn't it be relevant?
